apps/portals/src/portal-components/nf/BrowseToolsPage.tsx

```diff
@@ -218,14 +238,16 @@ const BrowseToolsPage = () => {
           </div>
         </div>
         <div className="center-content">
-          <Button
+          <WideButton
+            sx={wideButtonSx}
             href="https://forms.gle/htFkH5yewLzP1RAu7"
-            className="btn-wide highlightSubmitToolButton"
-            variant='contained'
+            className="highlightSubmitToolButton"
+            variant="contained"
+            // @ts-ignore - target prop exists, but TS doesn't recognize on styled component
```

Comment:
`target` prop not recognized in the styled component (WideButton). I tried adding `as typeof Button` to the styled component as recommended [here](https://github.com/mui/material-ui/issues/31314), but it didn't resolve the type error. However, component still functions and styled as expected, so ignoring this line for now.